torch/csrc/utils/tensor_numpy.cpp

```diff
@@ -157,6 +157,7 @@ static int aten_to_dtype(const ScalarType scalar_type) {
     case kChar: return NPY_INT8;
     case kByte: return NPY_UINT8;
     case kBool: return NPY_BOOL;
+    case kQInt8: return NPY_UINT8;
```

Comment:
why do we need this?  This seems like surprising behavior -- can we force the user to convert to int8 first and keep the existing code?

Reply:
This is needed when we want to call numpy() on a QTensor, we return a uint8_t so that users can inspect the values.

Reply:
I'd say numpy() on qtensor should rather fail all together. It's weird that .item() and .numpy() return different things. If you want values - call int_repr().numpy()

Reply:
are you saying we should return .item() should return a numpy object as well?

Reply:
item() returns Scalar for normal tensor right?

Reply:
Sure I will disable numpy() for QTensor